sys-fs/bees/files/v0.5-gentoo_build.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/makeflags b/makeflags
+index f5983cb..0348623 100644
+--- a/makeflags
++++ b/makeflags
```

Comment:
BTW if you're planning on submitting something upstream, you may want to convince them to split those variables more. Usually it's good idea to have separate variable for `-D -I` flags that should not be overriden, and e.g. `CFLAGS ?= -O3 -march...`, then combine them both on command-line.

Also, please make sure that the build system respects `CC`, `CXX`, `LDFLAGS` and preferably `CPPFLAGS` correctly.

Reply:
I'm still working on that part because I don't like that either. I'm pretty sure it will be accepted upstream. I'll try to get it in before v0.6 is tagged, so we can remove this.